run_dir/design/running_notes_tab.html

```diff
@@ -6,11 +6,25 @@
                 <div class="panel-heading">Add New Running Note</div>
                 <div class="panel-body">
                     <div class="row">
-                        <div class="col-md-6">
+                        <div class="col-md-7 form-inline">
+                          <label>Choose category:</label>
+                          <div class="btn-toolbar" data-toggle="buttons" style="margin-top:5px">
+                             <button class="btn btn-xs btn-info btnCatFilter" value="Decision" data-toggle="tooltip" title="Use this category when an executive decision has been made">Decision &nbsp;<span class="fa fa-thumbs-up"</span></button>
+                             <button class="btn btn-xs btn-success btnCatFilter" value="Lab" data-toggle="tooltip" title="Use this category for lab-related work">Lab &nbsp;<span class="fa fa-bong"</span></button>
+                             <button class="btn btn-xs btn-warning btnCatFilter" value="Bioinformatics" data-toggle="tooltip" title="Use this category for all bioinformatics work">Bioinformatics &nbsp;<span class="fa fa-laptop-code"</span></button>
+                             <button class="btn btn-xs btn-usr btnCatFilter" value="User Communication" data-toggle="tooltip" title="Use this category for notes influenced by user-contact">User Communication &nbsp;<span class="fa fa-people-arrows"</span></button>
+                             <button class="btn btn-xs btn-danger btnCatFilter" value="Administration" data-toggle="tooltip" title="Use this category for notes involving documentation">Administration &nbsp;<span class="fa fa-folder-open"</span></button>
+                             <button class="btn btn-xs btn-imp btnCatFilter" value="Important" data-toggle="tooltip" title="Use this category when a note needs to be highlighted">Important &nbsp;<span class="fa fa-exclamation-circle"</span></button>
+                             <button class="btn btn-xs btn-devi btnCatFilter" value="Deviation" data-toggle="tooltip" title="Use this category for notes about a deviation">Deviation &nbsp;<span class="fa fa-frown"</span></button>
+                          </div>
+                        </div>
+                    </div>
```

Comment:
I think there is one too many closing </div> tags here?

Reply:
But I might be wrong.

Reply:
See? Whitespace matters! 😆

Reply:
Don't think so, it looks correct to me in Atom, they all match one <div>